### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ilyaivasyk**
- [DEMO LINK](https://ilyaivasyk.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://ilyaivasyk.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
.

File: src/index.html [hunk 1]
```diff
@@ -7,8 +7,51 @@
...
+          <a href="#">
+            <img src="images/logo.png" alt="logo">
+          </a>
+        </div>
+        <div class="menu">
+          <nav class="nav-links">
+            <ul class="list">
+              <li>
+                <a href="#" class="is-active">Apple</a>
+              </li>
```
Comment (sTorba24) [inline src/index.html:29]:
```suggestion
              <li class="nav-item">
                <a href="#" class="nav-link is-active">Apple</a>
              </li>
```

Comment (sTorba24) [reply src/index.html:29]: fix everywhere

File: src/style.css [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+.list li a {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #000;
+  font-size: 12px;
+  text-transform: uppercase;
+  height: 100%;
+  font-weight: 500;
+}
```
Comment (sTorba24) [inline src/style.css:61]: don't style tags, use classes

Comment (sTorba24) [reply src/style.css:61]: then you also won't need such chain of selectors

File: src/style.css [hunk 3]
```diff
@@ -1,3 +1,80 @@
...
+
+.nav-links .list [data-qa="hover"]:hover {
+  color: var(--header-active-link-color);
+}
```
Comment (sTorba24) [inline src/style.css:65]: you should change color for all links
data-qa="hover" you need for tests, test will search item with such attribute and will hover it.

File: src/style.css [hunk 4]
```diff
@@ -1,3 +1,80 @@
...
+
+.nav-links .list .is-active {
```
Comment (sTorba24) [inline src/style.css:67]:
```suggestion
.is-active {
```

File: src/style.css [hunk 5]
```diff
@@ -1,3 +1,80 @@
...
+  position: relative;
+  color: var(--header-active-link-color);
+}
+
+.nav-links .list .is-active::after {
```
Comment (sTorba24) [inline src/style.css:72]:
```suggestion
.is-active::after {
```

**Review by sTorba24 — CHANGES_REQUESTED** [review]
Well done! Lets improve it

also check your demo link it looks broken

File: src/index.html [hunk 6]
```diff
@@ -7,8 +7,53 @@
...
+              <li class="nav-item">
+                <a class="nav-link" href="#">Samsung</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#">Smartphones</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#"
+                   data-qa="hover">Laptops & Computers
+                </a>
```
Comment (volodymyr-soltys97) [inline src/index.html:39]:
```suggestion
                <a
                  class="nav__link"
                  href="#"
                  data-qa="hover"
                >
                  Laptops & Computers
                </a>
```

**Pull request closed without merging**